ereuse_devicehub/resources/documents/templates/documents/wbConfiguration.ini

```diff
@@ -0,0 +1,14 @@
+DH_TOKEN='{{token}}'
```

Comment:
I suggest as an example of settings.ini file:

[settings]

DH_TOKEN="..."

DH_HOST="..."
DH_DATABASE="..."
DEVICEHUB_URL=https://${DB_HOST}/${DB_DATABASE}/


WB_BENCHMARK = False
WB_STRESS_TEST = 0
WB_SMART_TEST = ""

WB_ERASE = False
WB_ERASE_STEPS = 1
WB_ERASE_LEADING_ZEROS = False

WB_DEBUG = True